fiware-region-sanity-tests/commons/nova_operations.py

```diff
@@ -237,12 +237,17 @@ def delete_server(self, instance_id):
         """
         self.client.servers.delete(instance_id)
 
-    def list_servers(self):
+    def list_servers(self, name_prefix=None):
         """
         Gets all the servers of the tenant
+        :param name_prefix: Prefix to match server names
         :return: A list of :class:`Server`
         """
-        return self.client.servers.list()
+        server_list = self.client.servers.list()
+        if name_prefix:
+            server_list = [server for server in server_list if server.name.startswith(name_prefix)]
+
+        return server_list
```

Comment:
Just a comment, It should be applied in the same way to the rest of the resources. Shouldn't?